rapid7_insightvm_cloud/help.md

```diff
@@ -4,11 +4,13 @@ InsightVM is a powerful vulnerability management tool which finds, prioritizes,
 
```

Comment:
Up above, add a bit more details to descript:
`InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses the v4 cloud API to view assets, retrieve scan results and start scans. With an InsightVM scan engine paired to the platform, take advantage of triggering scans on assets in your environment along with performing robust asset searches to lookup and make actionable the asset and vulnerability data in your environment.`